client/components/backup-retention-management/loading/index.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
```

Comment:
Should we import only the `React.FC` type if we really need that only?

Reply:
Fixed!